src/eventstream.cr

```diff
@@ -0,0 +1,56 @@
+require "base64"
+
+module EventStream
```

Comment:
Can we wrap this in the `AWS` namespace? Also, is this a common AWS pattern? I haven't used any streaming APIs of theirs, so I'm not sure if it's Bedrock-only or something they use elsewhere.

Reply:
Yes to the namespace, good catch.

Let me see if I can dig up more info about this format and see if it's common

Reply:
S3Select uses it. Added a link to the doc explaining the format